libs/langchain/langchain/evaluation/parsing/base.py

```diff
@@ -122,9 +122,12 @@ def evaluation_name(self) -> str:
         return "json_equality"
 
     def _parse_json(
-        self, string: str
+        self,
+        string: Any,
```

Comment:
should/coudl we Union[str, dict] instead of Any?

Reply:
could be a list or a none or a lot of things too

Reply:
why cast as dict then?

Reply:
Ya but that's a larger PR. I should make generics for the parent class. Not sure the typing is actually useful in practice for these evaluators

Reply:
> why cast as dict then?

Ya I could cast as `Union[float, int, bool, None, dict, list]` instead 

At some point we ought to do with our other parsers too
```
from langchain.output_parsers.json import parse_json_markdown
parse_json_markdown("5")
# 5
```